Prototope/Gesture.swift

```diff
@@ -268,6 +268,79 @@ public class PanGesture: GestureType {
+    /** The handler will be invoked as the gesture recognizes and updates; it's passed the gesture's current
+    phase (see ContinuousGesturePhase documentation), rotation relative to the state at the beginning 
+    of the gesture in radians, rotation velocity in radians per second and also a touch sequence representing the center of all the touches involved in the rotation gesture. */
+    public convenience init(_ handler: (phase: ContinuousGesturePhase, rotationRadians: Double, rotationVelocity: Double, centroidSequence: TouchSequence<Int>) -> ()) {
+        self.init(handler: handler)
+    }
+    
+    /** 
+    
+    When cancelsTouchesInLayer is true, touches being handled via touchXXXHandlers will be cancelled
+    (and touch[es]CancelledHandler will be invoked) when the gesture recognizes.
+    
+    The handler will be invoked as the gesture recognizes and updates; it's passed the gesture's current
+    phase (see ContinuousGesturePhase documentation), rotation relative to the state at the beginning
+    of the gesture in radians, rotation velocity in radians per second and also a touch sequence representing the center of all the touches involved in the rotation gesture. */
+    public init(cancelsTouchesInLayer: Bool = true, handler: (phase: ContinuousGesturePhase, rotationRadians: Double, rotationVelocity: Double, centroidSequence: TouchSequence<Int>) -> ()) {
+        rotationGestureHandler = RotationGestureHandler(actionHandler: handler)
+        rotationGestureRecognizer = UIRotationGestureRecognizer(target: rotationGestureHandler, action: "handleGestureRecognizer:")
+        rotationGestureRecognizer.cancelsTouchesInView = cancelsTouchesInLayer
+    }
+    
+    private let rotationGestureRecognizer: UIRotationGestureRecognizer
+    private let rotationGestureHandler: RotationGestureHandler
+    
+    public weak var hostLayer: Layer? {
+        didSet { handleTransferOfGesture(rotationGestureRecognizer, oldValue, hostLayer) }
+    }
+    
+    deinit {
+        rotationGestureRecognizer.removeTarget(rotationGestureHandler, action: "handleGestureRecognizer:")
+    }
+    
+    @objc class RotationGestureHandler: NSObject {
+        private let actionHandler: (phase: ContinuousGesturePhase, rotationRadians: Double, rotationVelocity: Double, centroidSequence: TouchSequence<Int>) -> ()
+        private var centroidSequence: TouchSequence<Int>?
+        
+        init(actionHandler: (phase: ContinuousGesturePhase, rotationRadians: Double, rotationVelocity: Double, centroidSequence: TouchSequence<Int>) -> ()) {
+            self.actionHandler = actionHandler
+        }
+        
+        func handleGestureRecognizer(gestureRecognizer: UIGestureRecognizer) {
+            let rotationGesture = gestureRecognizer as UIRotationGestureRecognizer
+            switch rotationGesture.state {
+            case .Began:
+                // Reset the gesture to record rotation relative to the starting rotation
```

Comment:
Hm, but the following line doesn't actually reset the gesture if I understand correctly.